test_suite/tests/test_annotations.rs

```diff
@@ -643,6 +643,352 @@ fn test_unknown_field_rename_struct() {
     );
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename = "IgnoredRename")]
+struct IntRenameStruct {
```

Comment:
I think it will work as is, but it might be nice to have a test for renames to hex/binary/octal just in case.